pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -15,17 +17,32 @@ import (
 )
 
 const (
-	cdiAPIVersion = "cdi.kubevirt.io/v1alpha1"
+	cdiAPIVersion           = "cdi.kubevirt.io/v1alpha1"
+	customPropertyHugePages = "hugepages"
+	// LabelOrigin define name of the label which holds value of origin attribute of oVirt VM
+	LabelOrigin = "origin"
+	// LabelInstanceType define name of the label which holds value of instance type attribute of oVirt VM
+	LabelInstanceType = "instanceType"
+	// LabelTag define name of the label which holds tags of oVirt VM
+	LabelTag = "tags"
+	// LabelFQDN define name of the label which holds fqdn of oVirt VM
+	LabelFQDN = "fqdn"
```

Comment:
should we map the `fqdn` to hostname field of  https://kubevirt.io/api-reference/v0.26.4/definitions.html#_v1_virtualmachineinstancespec ?